mobly/controllers/android_device_lib/snippet_client_v2.py

```diff
@@ -89,11 +89,12 @@ class Config:
       other purposes may not take effect and you should use snippet RPCs. This
       is because Mobly snippet runner changes the subsequent instrumentation
       process.
+    user_id: The user id under which to launch the snippet process.
   """
 
   am_instrument_options: Dict[str, str] = dataclasses.field(
       default_factory=dict)
-
+  user_id: Union[int, None] = None
```

Comment:
Should we just use `Optional`?

Reply:
> Should we just use `Optional`?

No, because that is new in 3.10 but we officially support 3.7 / 3.8 / 3.9 too in I.